assets/dev/js/frontend/handlers/container/grid-container.js

```diff
@@ -221,4 +229,15 @@ export default class GridOutline extends elementorModules.frontend.handlers.Base
 		window.removeEventListener( 'elementor/editor/element-rendered', this.lifecycleChangeListener );
 		window.removeEventListener( 'elementor/editor/element-destroyed', this.lifecycleChangeListener );
 	}
+
+	updateEmptyViewHeight() {
+		const { emptyView } = this.elements,
+			{ grid_rows_grid: gridRows } = this.getElementSettings();
```

Comment:
@davseve Because it is a responsive control, should you not get the value for the current device?

Reply:
Changed, Added the current device